src/io/flutter/view/FlutterViewToggleableAction.java

```diff
@@ -42,7 +42,7 @@ public final void update(@NotNull AnActionEvent e) {
 
   @Override
   public void actionPerformed(AnActionEvent event) {
-    if (view.getFlutterApp() == null) {
+    if (app == null) {
```

Comment:
And here, I think we can get rid of this test since `app` should never be null?

Reply:
done